index.md

```diff
@@ -15,13 +15,13 @@ heading: Get Started
 This tutorial describes how to install and start using Kendo UI for Angular.
 
 * First, we will go through the installation steps.
-* Next, we'll create a simple app with a few Kendo UI Angular components. The [app source code](https://github.com/telerik/kendo-angular-getting-started) is available for your reference.
+* Next, we'll create a simple app with a few Kendo UI Angular for components. The [app source code](https://github.com/telerik/kendo-angular-getting-started) is available for your reference.
 * Finally, you will learn how to activate your Kendo UI for Angular license.
-* At all steps, we will recommend relevant learning resources that will help you be successful with Kendo UI for Angular.
+* On every step, we will recommend relevant learning resources that will help you be successful with Kendo UI for Angular.
 
 <blockquote class="alert-note">
     <h3 style="margin-top: 0;">Do You Prefer Video?</h3>
-    <p>You can setup an Angular project and add Kendo UI Angular components by following the steps below. Alternatively, you can check our <a href="https://www.telerik.com/campaigns/kendo-ui/using-kendo-ui-with-angular-video-tutorial">Angular video tutorials</a> or our <a href="https://www.telerik.com/kendo-angular-ui/components/virtual-class/">full-blown Kendo UI Angular video course</a>.</p>
+    <p>You can setup an Angular project and add Kendo UI for Angular components by following the steps in this guide. Alternatively, you can check our <a href="https://www.telerik.com/campaigns/kendo-ui/using-kendo-ui-with-angular-video-tutorial">Angular video tutorials</a> or our <a href="https://www.telerik.com/kendo-angular-ui/components/virtual-class/">full-blown Kendo UI Angular video course</a>.</p>
 </blockquote>
 
 ## Set up the Angular Project
```

Comment:
Is this note the first thing that I need to know before I start? Or can we move this note down?
`The latest Kendo UI for Angular packages target the current <a href="https://angular.io/guide/releases#support-policy-and-schedule">Angular long-term support versions</a> up to the latest official Angular version.
`

Reply:
The idea behind this note is that the customer must ensure they have an appropriate version of the Angular CLI, before installing Kendo UI Angular packages. On a second thought, I guess we could provide a bit more details about this, move the content to System Requirements and link to it. What do you think?